pages/common/piper.md

```diff
@@ -0,0 +1,28 @@
+# piper
+
+> A fast, local neural text to speech system that sounds great.
+> More information: <https://github.com/rhasspy/piper>.
+
+- Output a wav [f]ile using a [m]odel (assuming config json at path to model_path + .json):
+
+`echo {{Thing to say}} | piper -m {{path/to/model.onnx}} -f {{outputfile.wav}}`
+
+- Output a wav [f]ile using a [m]odel and specifying its [c]onfig json:
+
+`echo {{Thing to say}} | piper -m {{path/to/model.onnx}} -c {{path/to/model.onnx.json}} -f {{outputfile.wav}}`
+
+- Select a particular speaker in a voice with multiple speakers:
```

Comment:
```suggestion
- Select a particular speaker in a voice with the specified number of speakers:
```

Reply:
This is incorrect, so I think I need to reword it since it's clear my original wording was confusing. Voices have 1+ "speakers", and the specified number chooses the speaker, not the number of speakers.

So for example the thorsten_emotional voice has the following speaker options:
```
amused (0)
angry (1)
disgusted (2)
drunk (3)
neutral (4)
sleepy (5)
surprised (6)
whisper (7)
```
and the number selects which speaker you're choosing. Specifying the number of speakers doesn't really make sense in context.

Do you have any suggestions how I could make this clearer, succinctly?